yargs/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
I recommend using 2 space indent for JSON and Yaml file, because `package.json` use 2 space indent (if you use other indent style, some tools throw exceptions.)

And, `trim_trailing` and `insert_final_newline` rules should be inherited the root config.

I suggest to cover only `*.ts` rule and remove two rules.